packages/fxa-admin-panel/pm2.config.js

```diff
@@ -2,35 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const { resolve } = require('path');
+
 module.exports = {
   apps: [
     {
       name: 'admin',
-      cwd: __dirname,
-      script: 'ts-node -P server/tsconfig.json server/bin/fxa-admin-panel.ts',
+      cwd: resolve(__dirname, 'server'),
+      script: 'node -r ts-node/register bin/fxa-admin-panel.ts',
       max_restarts: '1',
       min_uptime: '2m',
       env: {
         LOGGING_FORMAT: 'pretty',
         NODE_ENV: 'development',
         NODE_OPTIONS: '--inspect=9140',
         PROXY_STATIC_RESOURCES_FROM: 'http://localhost:8092',
-        CONFIG_FILES: 'config/secrets.json',
+        CONFIG_FILES: '../config/secrets.json',
         PORT: '8091',
       },
+      filter_env: ['npm_'],
     },
     {
       name: 'admin-react',
       cwd: __dirname,
-      script: 'rescripts start',
+      script: 'yarn rescripts start',
       max_restarts: '1',
       min_uptime: '2m',
       env: {
+        SKIP_PREFLIGHT_CHECK: 'true',
         NODE_ENV: 'development',
         PUBLIC_URL: 'http://localhost:8091',
         BROWSER: 'NONE',
         PORT: '8092',
+        PATH: process.env.PATH.split(':')
+          .filter(p => !p.includes(process.env.TMPDIR))
+          .join(':'),
```

Comment:
yarn sets up a sandbox dir to run tasks and sets PATH and some other env vars to use it, but by the time pm2 starts the script the dir is gone, so this cleans up the path to remove the dead dir.